scripts/add-event.js

```diff
@@ -1,23 +1,22 @@
 var firebase = require("firebase/app");
 require("firebase/auth");
 require("firebase/firestore");
-var read = require("read");
 
 function usage() {
   console.log(`
 ${process.argv[1]}: Add an event to a venue
 
 Specify the name, time, duration in minutes, description, collective price, and price.
 
-Usage: node ${process.argv[1]} API_KEY PROJECT_ID VENUE UTC_TIME DURATION_MINUTES PRICE COLLECTIVE_PRICE NAME DESCRIPTION
+Usage: node ${process.argv[1]} API_KEY PROJECT_ID VENUE UTC_TIME DURATION_MINUTES PRICE COLLECTIVE_PRICE NAME DESCRIPTION USERNAME PASSWORD
 
-Example: node ${process.argv[1]} aaazzz111222333 co-reality-map examplevenue 2020-08-01T19:00:00Z 180 500 2500 "The Virtual Jazz Bar" "Join us in Virtual jazz bar on Saturday for three sets of fabulous jazz from the House band."
+Example: node ${process.argv[1]} aaazzz111222333 co-reality-map examplevenue 2020-08-01T19:00:00Z 180 500 2500 "The Virtual Jazz Bar" "Join us in Virtual jazz bar on Saturday for three sets of fabulous jazz from the House band." username password
 `);
```

Comment:
Instead of adding username/password fields to the params like this (which could be a security concern, as they are then going to be logged in the command history in your shell), it might be a better idea to refactor this script to use the firebase admin SDK + service account key pattern that most of our current/modern scripts use. This would also allow us to clean up/simplify a bunch of the patterns used here at the same time.

At the same time, we could also change this to a `.ts` file and rewrite it to benefit from TypeScript types rather than being plain JavaScript.